source/manuals/logit-io.html.md.erb

```diff
@@ -266,24 +226,26 @@ Declare an action on matching messages that uses the `omfwd` module to forward t
 )
 ```
 
+```
```

Comment:
I don't see why you've added this formatting?  This paragraph looks like it should be normal text, not a code sample.

Reply:
The idea was to highlight the 'Note' but I agree it doesn't work, this format should just be for code snippets.